core/modfunc.go

```diff
@@ -435,6 +443,73 @@ func (fn *ModuleFunction) linkDependencyBlobs(ctx context.Context, cacheResult *
 	return nil
 }
 
+// func (fn *ModuleFunction) applyIgnoreOnDir(ctx context.Context, dag *dagql.Server, arg *FunctionArg, value any) (any, error) {
```

Comment:
We should avoid committing commented code - it simply ends up getting out of date with the rest of the code.

Reply:
Yeah, I'll remove it

Reply:
I removed the commented code in [d974dcb](https://github.com/dagger/dagger/pull/8436/commits/d974dcbd73cbb7d23e947bc087577d451a75216e)